Dockerfile

```diff
@@ -2,7 +2,7 @@
 ARG TAG=local
 
 # This requires access to the Azure Container Registry
-FROM cfaprdbatchcr.azurecr.io/cfa-epinow2-pipeline-dependencies:${TAG}
+FROM cfaprdbatchcr.azurecr.io/cfa-epinow2-pipeline:${TAG}
```

Comment:
Can you explain this part? The dependencies image no longer has `-dependencies` in the name? I thought the main image was called `cfaprdbatchcr.azurecr.io/cfa-epinow2-pipeline:${TAG}`

Reply:
Because the `dependencies` value is passed implicitly in the second container build: https://github.com/CDCgov/cfa-actions/blob/9390790487d5013d0ddc1e6820c347fd323ad480/twostep-container-build/action.yml#L225